client/src/layouts/Game/Game.jsx

```diff
@@ -46,7 +47,7 @@ export const Game = ({ showDebug }) => {
     const players = useSelector(playersListSelector);
     const options = useSelector(gameSettingsSelector);
     const gameID = useSelector(gameIdSelector);
-    const playerID = useSelector(playerIdSelector);
+    //const playerID = useSelector(playerIdSelector);
```

Comment:
pls return mi code :(((( I will use it one day maybe

Reply:
actually this is fine

Reply:
i remove row completely :)))))